sys-apps/smartmontools/smartmontools-6.4-r1.ebuild

```diff
@@ -85,14 +87,14 @@ pkg_postinst() {
 		local db_path="/var/db/${PN}"
 
 		if [[ -f "${db_path}/drivedb.h" ]] ; then
-			ewarn "WARNING! The drive database file has been replaced with the version that"
-			ewarn "got shipped with this release of ${PN}. You may want to update the"
+			ewarn "WARNING! The existing copy of the drive database has been replaced with the version that"
+			ewarn "was shipped with this release of ${PN}. You may want to update the"
 			ewarn "database by running the following command as root:"
 			ewarn ""
-			ewarn "/usr/sbin/update-smart-drivedb"
+			ewarn "${UPDATE_DRIVEDB}"
```

Comment:
This prints `.../files/smartmontools-update-drivedb` instead of the installed path, which I don't think you intended.